app/views/components/_vacancy_item.html.erb

```diff
@@ -0,0 +1,29 @@
+<%#
+Vacancy Item
+
+Provide a card containing the informations of the job and when it is clicked, it gets redirected to the URL specified in the parameter.
```

Comment:
I guess it sounds better:
`Provide a card containing job's information, when clicked it gets redirected...`

plus, information is uncountable, no plural.